sorting/comb_sort.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#define SHRINK 1.3	//suggested shrink factor value
+
+void sort (int *numbers, int size) 
+{
+	int gap = size;
+	while (gap > 1) 	//gap = 1 means that the array is sorted 
+	{
+		gap = gap/SHRINK;
+		int i = 0;
+		while ((i + gap) < size) 
+		{	//similiar to the Shell Sort
+			if (numbers[i] > numbers[i + gap])
+			{
+				int tmp = numbers[i];
+				numbers[i] = numbers[i + gap];
+				numbers[i + gap] = tmp;
+			}
+			i++;
+		}
+	}
+}
+int main()
```

Comment:
Please add a newline between the begin of the main-function and the code line above.